File: storydoc-backend/src/main/java/io/storydoc/server/code/app/SourceCodeConfigDTO.java
package io.storydoc.server.code.app;

import io.storydoc.server.code.domain.SourceCodeConfigId;
import io.storydoc.server.storydoc.app.dto.StoryDocSummaryDTO;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;

import java.util.List;

@Data
@AllArgsConstructor
@Builder
public class SourceCodeConfigDTO {
    private SourceCodeConfigId id;
    private List<String> dirs;
    private String name;
    private StoryDocSummaryDTO storyDocSummary;

}

File: storydoc-frontend/src/app/api/models/source-code-config-dto.ts
/* tslint:disable */
/* eslint-disable */
import { SourceCodeConfigId } from './source-code-config-id';
import { StoryDocSummaryDto } from './story-doc-summary-dto';
export interface SourceCodeConfigDto {
  dirs?: Array<string>;
  id?: SourceCodeConfigId;
  name?: string;
  storyDocSummary?: StoryDocSummaryDto;
}
